libcst/matchers/_matcher_base.py

```diff
@@ -229,14 +316,18 @@ def __repr__(self) -> str:
         return f"MatchIfTrue({repr(self.func)})"
 
 
-def MatchRegex(
-    regex: Union[str, Pattern[AnyStr]]
-) -> MatchIfTrue[Callable[[str], bool]]:
+def MatchRegex(regex: Union[str, Pattern[str]]) -> MatchIfTrue[Callable[[str], bool]]:
     """
-    Used as a convenience wrwapper to :class:`MatchIfTrue` which allows for
-    matching against a regex. This should be used against attributes which
-    are strings, to allow matching regular expressions. An example would be
-    ``m.Name(m.MatchRegex(r'[A-Za-z]+'))``.
+    Used as a convenience wrapper to :class:`MatchIfTrue` which allows for
```

Comment:
It worth mentioned that MatchRegex uses Python builtin `re` module (so special sequences is available e.g. \d \w) and link to re doc https://docs.python.org/3/library/re.html. 
Also mentions it takes either a regex string or a `Pattern`.

Reply:
Can do.